<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="B_Register">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Register New User</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="I_Password">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="L_password">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PassWord</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_UserName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>User Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="I_UserName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="L_passwordRe">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Repeat PassWord</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="I_PasswordRe">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
